src/Catalog/LogEvents.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGet.Services.Metadata.Catalog
+{
+    public static class LogEvents
+    {
+        public static EventId ValidationFailed = new EventId(900, "Validation failed!");
```

Comment:
>900 [](start = 61, length = 3)

Why are we using explicit event IDs here? Not necessarily a bad thing just curious since we don't use 'em much elseware I think.